packages/webhooks/lib/auth.ts

```diff
@@ -45,11 +45,17 @@ export async function sendAuth({
         return;
     }
 
-    if (operation !== 'creation' && operation !== 'refresh') {
+    if (operation === 'unknown') {
         return;
     }
 
-    if (!shouldSend({ success, type: operation === 'creation' ? 'auth_creation' : 'auth_refresh', webhookSettings })) {
+    if (
+        !shouldSend({
+            success,
+            type: operation === 'creation' ? 'auth_creation' : operation === 'override' ? 'auth_override' : 'auth_refresh',
+            webhookSettings
+        })
+    ) {
```

Comment:
I personally have a hard time parsing double ternary operator. Maybe that's just me but I would prefer a switch or a Map 
```
const getAuthType = (op: string) => {
  switch (op) {
    case 'creation': return 'auth_creation';
    case 'override': return 'auth_override';
    default: return 'auth_refresh';
  }
};

...
type: getAuthType(operation)
...
```

Reply:
or a map yes